### ISSUE TITLE
Proposal: Require that scramble sequences for one round are only used on one calendar day

### ISSUE THREAD

**Issue by Laura-O** [thewca/wca-regulations#295]
There are competitions where 4/5 BLD can be done on two consecutive days.
I am not sure if this is done likewise in other parts of the world, so I will explain it briefly: when a competitor wants to start an attempt he finds someone to judge him, informs the delegate and submits his puzzle. The judge receives the scrambled puzzle and the scoresheet. The competitor and the judge can then start the attempt (usually in a side room).

This is generally a good approach as it makes it possible to do these unpopular events without blocking a specific time slot. BigBlinders also like it because they can choose when they want to compete and have longer pauses between their attempts. However, delegates have to be very careful and always keep an eye on who is judging and that all regulations are followed.
As a down side this approach gives competitors some additional ways to cheat. In a worst case situation a competitor uploads a video of a solve on the first day of the competition and another competitor analyses the solve, practices the memo and does the attempt on the next day. Hypothetically he could have more than 24 hours to do this.

I know that some delegates already do so, but I want to suggest to add a paragraph to the regulations that a scramble sequence for a round may only be used on one calendar day. This would significantly reduce possible ways of cheating. 
This could be either added as [4b4](https://www.worldcubeassociation.org/regulations/#4b) or added to [4b2](https://www.worldcubeassociation.org/regulations/#4b2).

**Comment by lgarron** [comment]
I agree that this is sensible, and already seems to be done in practice.

However, what are the particular ways that this reduces cheating compared
to using the same scramble over a large range of hours during a single day?
Is it that competitors are more likely to discuss scrambles at the end of
the day/online over a 24-hour period? Chance of someone uploading a video
that another competitor might see?

**Comment by Laura-O** [comment]
Well, basically I can think of two obvious ways to cheat: either there is an accomplice who shares his memo or someone shares a video of his solve (this might also happen without the intention to help someone to cheat).

Case 1 would mean that one competitor does his attempt, writes down the memo, hands it over to the second competitor and he memorizes it. This can easily be done in less than 30 minutes so this is not a particular problem in 2-day competitions, where the same scrambles are used on both days.

For case 2 a competitor definitely needs more time. I can't estimate how long it takes to do a 4x4/5x5 reconstruction but as long as you are not Brest (see [here](https://youtu.be/i7WEaEIXEmE)) it's probably more than one hour. Furthermore, one would have to leave the venue in order to avoid that someone notices what he is doing. Having a free evening/night makes that a lot easier.
Besides, in the unintentional case (without having an accomplice), it's unlikely that competitors upload videos during the day. It's a lot more likely that they do this after the first competition day. Actually I was reminded of this topic when I saw that a competitor had uploaded some solves on YouTube in the evening of the first competition day and I knew that he had also recorded his 4BLD attempts at other competitions.

To summarize this: more time for BLD attempts means more time for cheating. Having a competition pause in this time span (i.e. the time between two competition days) makes it also easier to prepare solves without anyone else noticing.

P.S.: I actually have some more thoughts on how the use of the same scrambles for two days can make cheating easier, but I don't want this to be some kind of cheating tutorial...

**Comment by Claster** [comment]
Proposal:
If 4x4/5x5 bld event has no specific time, rather competitors can do their attempts whenever they want, organizers should consider having different scrambles for everyone.
Since the number of competitors in these events is not so high, it is doable imho.

**Comment by pedrosino** [comment]
I haven't thought about this yet, but your reasoning makes sense.
I don't know or haven't been to a competition with 4/5bld happening in more than one day, but even in a single day it can be problematic. Maybe change scrambles after some time during the event window, but that doesn't seem fair since it's a final round.

**Comment by Laura-O** [comment]
I am still not sure about a reasonable amount of time and how this could be implemented in the regulations.

Furthermore, I realized that having a regulation for **rounds** would not be useful. Rounds in Fewest Moves and also MultiBLD are often held on two days with each attempt done individually, so there is no need to generate another set of scrambles.
This regulation should rather forbid that the same scramble sequence is used for an attempt on Saturday morning and 30 hours later on Sunday evening.

Not perfect yet, just an idea:
_Each scramble sequence must be applied during a maximum time frame of X hours. This time frame begins when the scramble sequence is used for the first time._

**Comment by Claster** [comment]
I like this idea (X=1 or 2 seems reasonable to me), though there might be some exceptions at the discretion of the delegate, so "should" instead "must".
A downside could be that if competitor A solves 4bld/5bld either faster or slower than his average, then competitor B can choose to either compete now or wait for the next scramble slot.

**Comment by viroulep** [comment]
Done in #342.

For the "final round fairness" point, I think the organizer can manage to get all the top people to solve in the same time frame.

**Comment by KitClement** [comment]
Per this discussion on speedsolving: 

https://www.speedsolving.com/forum/threads/one-answer-wca-competition-and-regulations-question-thread.50221/page-28#post-1202145

It seems to me that this "should" in 4b4 should really be a "must," unless there are reasons I'm not aware of for making this mandatory. Given that it is not required, it seems more of a guideline, and one can interpret with 1h++ (as Goosly did) that all competitors using the same scrambles is more important than the time frame.

**Comment by Claster** [comment]
We could clarify that 4b4 has a higher priority than 1h++. I don't think we should replace "should" with "must" in 4b4, at least not before we actually ask delegates what are the reasons they violate 4b4.

I can think of a couple examples where I could tolerate violating 4b4:
1) A scramble is applied within 2 hours and 1 minute;
2) I trust all competitors and they are not going to share the scrambles (e.g. there are two competitors, and one of them is me).
Both examples are theoretical though, and in both of them we could use two sets of scrambles just in case.

**Comment by Jambrose777** [comment]
Wouldn't the fact that the only available scramblers are all competing in the round be a reason to have 2 heats, especially for smaller competitions?

On Oct 31, 2016, at 7:04 AM, KitClement <notifications@github.com<mailto:notifications@github.com>> wrote:

Per this discussion on speedsolving:

https://www.speedsolving.com/forum/threads/one-answer-wca-competition-and-regulations-question-thread.50221/page-28#post-1202145

It seems to me that this "should" in 4b4 should really be a "must," unless there are reasons I'm not aware of for making this mandatory. Given that it is not required, it seems more of a guideline, and one can interpret with 1h++ (as Goosly did) that all competitors using the same scrambles is more important than the time frame.

##
